mssqlcli/jsonrpc/contracts/tests/test_json_rpc_contracts.py

```diff
@@ -173,8 +173,6 @@ def verify_connection_service_response(self,
                 elif isinstance(response, connectionservice.ConnectionCompleteEvent):
                     if response.connection_id:
                         complete_event += 1

```

Comment:
can you assert the connection_id here?

Reply:
nvm

---
In reply to: [168636959](https://github.com/dbcli/mssql-cli/pull/169#discussion_r168636959) [](ancestors = 168636959)